src/test/user.spec.js

```diff
@@ -193,3 +200,226 @@ describe('Test user login, signup and account verification', () => {
     });
   });
 });
+
+describe('USER CAN FOLLOW OTHER USERS', () => {
+  beforeEach(async () => {
+    await db.User.destroy({ cascade: true, truncate: true });
+  });
+
+  it('should be able to follow other users', async () => {
+    const firstUser = await createUser();
+    const secondUser = await createUser();
+    await createUser();
+    const token = jwt.sign(firstUser, process.env.JWT_SECRET);
```

Comment:
This method exists in already as a helper function. Kindly use that instead

Reply:
I have removed it now, thank you